### ISSUE TITLE
Zones Please

### ISSUE THREAD

**Issue by AlexKLMan** [stevenguh/homebridge-valetudo#1]
**Is your feature request related to a problem? Please describe:**
Yes, would love to clean rooms!

The silly thing cleans everything!

**Describe the solution you'd like:**
Tiles switches with each zone will be great.

**Describe alternatives you've considered:**

I tried Mqttt and can't get my head around it :)

**Additional context:**

Understand you're busy and can't work on this much, but thanks for reading!

**Comment by stevenguh** [comment]
This is something that bugs me from time to time as well. Since Apple just announced an official support for vacuum, probably will wait until before spending time on it.

**Comment by AlexKLMan** [comment]
Yeah I saw that with WWDC, ok no worries; I will check back in closer to the end of this year :) Thank you.

**Comment by 4o4R** [comment]
Checking in :)

**Comment by stevenguh** [comment]
Homebridge finally has Matter support! I've rolled out v2.0.2 of the plugin, which adds native robot vacuum support and segment control.
